store_app/views/external_API.py

```diff
@@ -0,0 +1,30 @@
+from flask import make_response, jsonify, g
+from app import app
+from tables.models import Movie
+
+
+@app.route('/movies/<int:movie_id>/description', methods=['GET'])
+def get_movie_description(movie_id):
+    try:
```

Comment:
Funny enough this try/catch pattern found everywhere (not bad at all, just a lot of code duplication) could also be achieved via a decorator function - assuming of course you are returning the same error everywhere. The big argument here is that in the end, you can come up with a uniform process to handle errors and return error codes for your API, and having a central place means you can adjust easily and have a global impact on your application. We refer to this as a "Global Exception Handler".

I expect there are perhaps even fancier ways to write something like a global exception handler in Flask that Nitin might know about, but I think even a decorator can make a difference.